File: CW/lesson_01/testapp/app/src/main/java/com/example/testapp/MainActivity.java
package com.example.testapp;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @SuppressLint("SetTextI18n")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        TextView textView = findViewById(R.id.textView);
        EditText edtiText = findViewById(R.id.ETName);
        Button btn = findViewById(R.id.confirm);

        btn.setOnClickListener(view ->{
            Toast.makeText(getApplicationContext(),"Hello, "+ edtiText.getText(),Toast.LENGTH_SHORT).show();
           textView.setText("Hello, "+ edtiText.getText());
        });
    }
}

File: CW/lesson_07/MVVM/app/src/main/java/com/example/mvvm/ui/activityes/MainActivity.kt
package com.example.mvvm.ui.activityes

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.mvvm.R
import com.example.mvvm.network.GlobalApi

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        GlobalApi.initialize()
    }
}